go-controller/hack/update-codegen.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if  ! ( command -v controller-gen > /dev/null ); then
+  # Need to take an unreleased version of controller-tools
+  # because of controller-tools bug 302
+  echo "controller-gen not found, installing sigs.k8s.io/controller-tools@83f6193..."
+  olddir="${PWD}"
+  builddir="$(mktemp -d)"
+  cd "${builddir}"
+  GO111MODULE=on go get -u sigs.k8s.io/controller-tools/cmd/controller-gen@83f6193
+  cd "${olddir}"
+  if [[ "${builddir}" == /tmp/* ]]; then #paranoia
+      rm -rf "${builddir}"
+  fi
+fi
+
+for crd in {egressfirewall,egressip}; do
```

Comment:
how about making this generic? something like 
```
crds=(NONE:$(ls pkg/crd) )
if [ ! -z ${crds} ];
 then exit
fi

for crd in ${crds}; do
```
that way just adding another directory into pkg/crd will cause the script to generate the files for it. it will also make this script functional for the time when one of out changes have merged

Reply:
Good remark. Will do.